sumcheck/src/verifier/mod.rs

```diff
@@ -147,3 +149,23 @@ pub enum SumCheckVerifierError {
     #[error("wrong opening point for the oracles")]
     WrongOpeningPoint,
 }
+
+// HELPER
+// =================================================================================================
+
+/// Evaluates periodic columns as multi-linear extensions.
+fn evaluate_periodic_columns_at<E: FieldElement>(
+    periodic_columns: PeriodicTable<E>,
+    eval_point: &[E],
+) -> Vec<E> {
```

Comment:
Should this be a method on the `PeriodicTable`?

Reply:
That was the initial plan but then the method depends on MultiLinear which is part of the sum-check crate. The later we can import from the air crate